utils/ANLEra_leader_options.cfg

```diff
@@ -487,10 +492,12 @@ Help an ally with their research"
                                 not_equals=ANLEra_Dwarves
                             [/variable]
 
-                            {AND_NOT_RECRUIT "Dwarvish Fighter"}
-                            {AND_NOT_RECRUIT "Dwarvish Guardsman"}
-                            {AND_NOT_RECRUIT "Dwarvish Thunderer"}
-                            {AND_NOT_RECRUIT "Dwarvish Ulfserker"}
+                            [and]
+                                {NOT_RECRUIT "Dwarvish Fighter"}
+                                {OR_NOT_RECRUIT "Dwarvish Guardsman"}
+                                {OR_NOT_RECRUIT "Dwarvish Thunderer"}
+                                {OR_NOT_RECRUIT "Dwarvish Ulfserker"}
+                            [/and]
```

Comment:
You don't need the [and] tags, and is the default.

Reply:
Right. Maybe this wasn’t the right way to do this, as I mostly wanted to group the `[or]`s together.

Reply:
Pardon, they are needed.